### PR TITLE
🐛 fix: Rust lang broken icon link

### PR DESCRIPTION
**Pull request by algovoid**
## What type of PR is this?

- [x] Bug Fix

## What I did

Changed the Rust language icon URL  as it does not work anymore and returns a `404`

### CODE REVIEW

**Review by maurodesouza — APPROVED** [review]

**Comment by maurodesouza** [comment]
Thanks @algovoid ❤

**Comment by algovoid** [comment]
@maurodesouza i have another PR waiting for approval can you please check it out . Thanks .

**Pull request merged**
